src/main/java/com/ivoyant/AssessmentBook/Controller/BookController.java

```diff
@@ -0,0 +1,44 @@
+package com.ivoyant.AssessmentBook.Controller;
+
+import com.ivoyant.AssessmentBook.Model.Book;
+import com.ivoyant.AssessmentBook.Service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class BookController {
+
+    @Autowired
+    private BookService service;
+
+    @PostMapping("/addBook")
+    public Book addBook( @RequestBody Book book) {
+        return service.saveBook(book);
+    }
+
+    @PutMapping("/Book")
+    public Book updateBook(@RequestBody Book book){
+        return service.upsert(book);
+    }
+
+    @GetMapping("/books")
+    public List<Book> getAllBooks(){
+        return service.getAllBooks();
+    }
+
+    @GetMapping("/books/{id}")
+    public Book getBookById(@PathVariable int id){
+        return  service.getBookById(id);
+    }
+
+    @DeleteMapping("/Book/{id}")
```

Comment:
It should be in lower case. 
"/book/{id}"

Reply:
changed it to lower case